src/HttpDriver/Configuration.php

```diff
@@ -3,31 +3,92 @@
 namespace GraphAware\Neo4j\Client\HttpDriver;
 
 use GraphAware\Common\Driver\ConfigInterface;
+use Http\Client\HttpClient;
+use Http\Discovery\HttpClientDiscovery;
+use Http\Discovery\MessageFactoryDiscovery;
+use Http\Message\RequestFactory;
 
 class Configuration implements ConfigInterface
 {
     /**
      * @var int
+     * @deprecated Will be removed in 5.0
      */
     protected $timeout;
 
     /**
      * @var string
+     * @deprecated Will be removed in 5.0
      */
     protected $curlInterface;
```

Comment:
Again, this class is for configuration and is not tight to guzzle in particular, we still need a way to be able to specify the network interface to use.